annotations/crd-annotations/src/main/java/io/dekorate/crd/decorator/AddAdditionPrinterColumnDecorator.java

```diff
@@ -18,9 +18,9 @@
 package io.dekorate.crd.decorator;
 
 import io.dekorate.utils.Strings;
-import io.fabric8.kubernetes.api.builder.Predicate;
-import io.fabric8.kubernetes.api.model.apiextensions.CustomResourceColumnDefinitionBuilder;
-import io.fabric8.kubernetes.api.model.apiextensions.CustomResourceDefinitionVersionFluent;
+import java.util.function.Predicate;
+import io.fabric8.kubernetes.api.model.apiextensions.v1beta1.CustomResourceColumnDefinitionBuilder;
```

Comment:
Should probably use `v1` version instead of `v1beta1`…

Reply:
At some point we should probably do that, but I think that for now we should keep the version we already used.
Going to `v1` is not a trivial task and will possibly cause lot of issues.